src/module/rules/CombatRules.ts

```diff
@@ -123,14 +121,34 @@ export class CombatRules {
         return modified;
     }
 
+    static isBlockedByVehicleArmor(incomingDamage: DamageData, attackerHits: number, defenderHits: number, actor: SR5Actor): boolean {
+        // TODO: Figure out why commenting out this block doesn't fail a test
+        if(!actor.isVehicle()) {
+            return false;
+        }
+
+        const modifiedDamage = CombatRules.modifyDamageAfterHit(actor, attackerHits, defenderHits, incomingDamage);
+
+        const modifiedAv = actor.getArmor(incomingDamage).value;
+        const modifiedDv = modifiedDamage.value;
+
+        console.log(`Checking vehicle armor - Modified AV: ${modifiedAv}, Modified DV: ${modifiedDv}`);
```

Comment:
I try to reduce usage of .log but I add plenty of console.debug

Reply:
I only see merit in risky or experimental code for debugs :) If you think this feature is one of these add all the logs you would need to trace errors.

Reply:
I feel confident enough in my tests here to omit this log entirely, but I'll keep debugs in mind for future changes!